### ISSUE TITLE
support other key hosts beyond github

### ISSUE THREAD

**Issue by max-mapper** [mafintosh/webcat#5]
right now we use github because: https://github.com/maxogden.keys

but we should be able to use any scheme where you can type a username and it can figure out how to get a public key. it should probably prompt the user and double check that they wanna trust the public key from the domain it resolves, similar to what keybase does.

keybase might be a good next one to support actually, as they have a CORS enabled endpoint to get the pgp key for any user

**Comment by max-mapper** [comment]
also re: github key use crypto, see this issue https://github.com/substack/cipherhub/issues/2 (we are using for signing and not encrypting)

**Comment by reqshark** [comment]
would having a lot of keys impact performance?

like https://github.com/reqshark.keys

**Comment by mafintosh** [comment]
@reqshark no. it just uses the first public key that matches a local private key

**Comment by guybrush** [comment]
what about keybase?

**Comment by max-mapper** [comment]
@guybrush did you read the first message in this thread?

**Comment by guybrush** [comment]
@maxogden lol sorry, somehow i did miss the last sentence or something was pretty late for me haha
